background.js

```diff
@@ -1,16 +1,29 @@
-const menuId = "detektor";
+const trackId = "detektor-track";
+const playlistId = "detektor-playlist";
 const storage = browser.storage.local;
 
 browser.contextMenus.create({
-    id: menuId,
+    id: trackId,
     title: "Add to Playlist"
 });
 
+browser.contextMenus.create({
+    id: playlistId,
+    title: "Analyze Entire Playlist"
+});
+
 browser.contextMenus.onClicked.addListener(function(info, tab) {
-    if (info.menuItemId == menuId) {
+    if (info.menuItemId == trackId) {
```

Comment:
This is kind of fuck, if the track finished and another one starts, this opens another window :p